### PR TITLE
Fix inconsistent QC metrics

### LINKED ISSUE #226: MS:4000072 has no parent
MS:4000072 has no parent and therefore it appears at the root of the ontology. What is the most suitable parent?

```
[Term]
id: MS:4000072
name: observed mass accuracy
def: "Observed mass accuracy in ppm, calculated by 1E6 x (observed m/z - theoretical m/z)/theoretical m/z of a selected and identified ion in a mass spectrum." [PSI:MS]
relationship: has_metric_category MS:4000014 ! single spectrum based metric
relationship: has_metric_category MS:4000008 ! ID based metric
relationship: has_metric_category MS:4000022 ! MS2 metric
relationship: has_units UO:0000169 ! parts per million
relationship: has_value_concept MS:1000014 ! accuracy
```

maybe it should have:
```
is_a: MS:4000003 ! single value
```

?

### LINKED ISSUE #227: MS:4000141 has no parent
MS:4000141 has no parent and therefore it appears at the root of the ontology. What is the most suitable parent?

```
[Term]
id: MS:4000141
name: outlier threshold criterion
def: "The definition of the outlier criteria applied." [PSI:MS]
```

### PR DESCRIPTION
**Pull request by bittremieux**
Fixes #226. Fixes #227.

### CODE REVIEW

File: psi-ms.obo [hunk 1]
```diff
@@ -23734,8 +23736,8 @@ name: MS2 precursor median m/z of identified quantification data points
 def: "Median m/z value for MS2 precursors of all quantification data points after user-defined acceptance criteria are applied. These data points may be for example XIC profiles, isotopic pattern areas, or reporter ions (see MS:1001805). The used type should be noted in the metadata or analysis methods section of the recording file for the respective run. In case of multiple acceptance criteria (FDR) available in proteomics, PSM-level FDR should be used for better comparability." [PSI:MS]
 comment: The m/z distribution informs about the dynamic range of the acquisition.
 is_a: MS:4000003 ! single value
-is_a: MS:4000008 ! ID based
-is_a: MS:4000020 ! ion source metric
+relationship: has_metric_category: MS:4000008 ! ID based
```
Comment (mobiusklein) [inline psi-ms.obo:23739]: I don't think relationship labels terminate with `:`

```
relationship: has_units UO:0000021 ! gram
relationship: has_value_type xsd:float ! The allowed value-type for this CV term
```

**Review by mobiusklein — CHANGES_REQUESTED** [review]
Minor syntax

**Review by mobiusklein — APPROVED** [review]

**Comment by mwalzer** [comment]
:+1:

**Review by edeutsch — APPROVED** [review]
Looks pretty good except that MS:4000008 is an "ID based metric", not an "ID based"

**Review by edeutsch — APPROVED** [review]

**Pull request merged**